clients/include/rocblas_random.hpp

```diff
@@ -110,6 +126,22 @@ inline int8_t random_generator<int8_t>()
     return std::uniform_int_distribution<int8_t>(1, 3)(rocblas_rng);
 };
 
+template <>
+inline rocblas_float_complex random_generator<rocblas_float_complex>()
+{
+    float real  = std::uniform_int_distribution<int>(1, 10)(rocblas_rng);
+    float image = std::uniform_int_distribution<int>(1, 10)(rocblas_rng);
+    return {real, image};
```

Comment:
`image` should be spelled `imag`.

`imag` is short for `imaginary`. There is no `e` in `imaginary`.

Reply:
understand. Thanks for catch this typo error. :)